op-supervisor/supervisor/backend/backend.go

```diff
@@ -163,6 +169,7 @@ func (su *SupervisorBackend) OnEvent(ev event.Event) bool {
 
 func (su *SupervisorBackend) AttachEmitter(em event.Emitter) {
 	su.emitter = em
+	su.rewinder.AttachEmitter(em)
 }
```

Comment:
Can we instead attach the rewinder to the event system as its own module?

I.e. `eventSys.Register("rewinder", su.rewinder, event.DefaultRegisterOpts())`
That will then call the `AttachEmitter` func on the rewinder.